avocado/plugins/run.py

```diff
@@ -100,34 +100,11 @@ def configure(self, parser):
             short_arg="-p",
         )
 
-        help_msg = (
```

Comment:
Any reason for removing this help message?

Reply:
Actually no, good catch.